src/sticker-builder/stickers/mention.sticker.ts

```diff
@@ -3,9 +3,9 @@ import { Enumerable } from '../../decorators';
 
 export class MentionSticker extends InstaSticker {
   public userId: string;
-  public displayType: 'mention_username' | 'mention_reshare' = 'mention_username';
+  public displayType?: 'mention_username' | 'mention_reshare' = 'mention_username';
   @Enumerable(false)
```

Comment:
BTW, why do we need it here?

Reply:
The mention-sticker is both used when tagging a user and resharing a story one was mentioned in. But here this defaults to the normal mention (e.g. by typing the username). The reshare would need the `reshared_media_id` to be set.

Reply:
I'm talking about `@Enumerable(false)`, github displays it wrong :)

Reply:
This is there so it doesn't get serialized because I only need need it in the additional properties.